docs/tutorial/05-creating-modules.md

```diff
@@ -168,8 +168,27 @@ Since the `targetScope` has been changed, notice that the `$schema` property in
 
 Modules are a powerful way to separate your bicep files into logical units and abstract away complex resource declarations.
 
+## Using if condition with bicep Modules
+Modules also support the if condition (introduced in v0.2.212). The linked ARM template that will be created by `bicep build` will insert a condition at the deployment level for everything in the module. To take advantage of this the module declaration needs to follow the if condition syntax:
```

Comment:
Nit: The linked ARM template ~~that will be~~ created by

Reply:
Updated to say "The linked ARM templated produced by..."